test/jdk/java/foreign/callarranger/TestSysVCallArranger.java

```diff
@@ -145,66 +145,6 @@ public void testNestedUnion() {
         assertEquals(bindings.nVectorArgs(), 0);
     }
 
-    @Test
-    public void testNestedStructsUnaligned() {
```

Comment:
These tests are no longer required, since they refer to a situation where the one of the layout involved would have an alignment different from the natural alignment (which is now enforced at link time).